spec/fake_braintree/customer_spec.rb

```diff
@@ -114,6 +114,18 @@
     expect(billing_address.street_address).to eq '1 E Main St'
     expect(billing_address.postal_code).to eq '60622'
   end
+
+  it "sets the creation time" do
+    customer = Braintree::Customer.create(
+      credit_card: {
+        number: TEST_CC_NUMBER,
+        expiration_date: '04/2016'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.